packages/fxa-auth-server/lib/senders/emails/templates/postAddTwoStepAuthentication/en-US.ftl

```diff
@@ -0,0 +1,6 @@
+postAddTwoStepAuthentication-subject = Two-step authentication enabled
+postAddTwoStepAuthentication-title = Two-step authentication enabled
```

Comment:
This could be `{ postAddTwoStepAuthentication-subject }` for less needed translations (I think that's how that works?)